### ISSUE TITLE
Does not render SVG

### ISSUE THREAD

**Issue by sedubois** [Dreamersoul/administrate-field-active_storage#36]
I have a model with `has_one_attached :logo`. I configured the corresponding Administrate dashboard with `logo: Field::ActiveStorage`, it allows to upload an SVG (it created a blob of content-type "image/svg+xml") but it does not render correctly:

<img width="189" alt="Screenshot 2020-04-03 at 14 37 27" src="https://user-images.githubusercontent.com/4217871/78361493-c4898300-75b8-11ea-817f-4f366f9aa534.png">

Tested with administrate-field-active_storage 0.2.2.

**Comment by sedubois** [comment]
OK the SVGs render correctly when adding this initializer:
```rb
# Warning: Make sure to sanitize SVGs if users gain the ability to upload themselves:
# https://github.com/rails/rails/issues/34665#issuecomment-446601748
Rails.application.config.active_storage.content_types_to_serve_as_binary.delete("image/svg+xml")
```

There is still one issue that the option `show_preview_size` has no effect for SVGs. In this case I guess it should be used to set the max width and height of the HTML `img` tag.

**Comment by Dreamersoul** [comment]
i believe this is related to the active_storage gem in general, maybe we could have part of the readme explain how to deal with svg content, i remember i faced some issues trying to attach an svg to a model while trying to inline it, so im not very knowledgeable here

**Comment by sedubois** [comment]
Yes that's what https://github.com/rails/rails/issues/34665 explains, maybe the README could have a reference to it. Also the render template would need to have a width/height for the SVG.

**Comment by Dreamersoul** [comment]
Perfect, I'm thinking of doing that after I merge the pending pull requests. Thank you for the continuous contributions (y)

**Comment by Dreamersoul** [comment]
i have opened a page with svg and it rendered normally can you double check?

**Comment by sedubois** [comment]
@Dreamersoul I haven't understood your question, could you rephrase? I confirm SVGs render correctly with the procedure in https://github.com/Dreamersoul/administrate-field-active_storage/issues/36#issuecomment-608446819.

BTW for sanitizing SVGs I came across the {active_storage_svg_sanitizer](https://github.com/hopsoft/active_storage_svg_sanitizer/) gem. 

As you suggested maybe the README could have a section mentioning those 2 things (sanitizing + rendering SVGs).

**Comment by Dreamersoul** [comment]
I was saying that svgs rendered perfectly in my own web app maybe I have the configuration on and forgot about it, adding the configuration to the readme would be certainly helpful!

**Comment by Dreamersoul** [comment]
is this still an issue? can we fix it?

**Comment by sedubois** [comment]
@Dreamersoul there is no issue, I was just lacking documentation. Hopefully this issue can serve as documentation for others who need to render SVGs.
